### PR TITLE
BCMOHAD-15762||Mass Assign

### PR DESCRIPTION
**Pull request by ahshaik1**

### CODE REVIEW

**Review by cgijeffolsen — APPROVED** [review]

File: TPL/force-app/main/default/classes/AccountQueueable.cls [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+            activeBatch = this.hccupdateList;
+        }
+        System.debug(LoggingLevel.INFO, 'Updating ' + activeBatch.size() + ' ' + sObjectType + ' records.');
+        Database.saveResult[] sr = database.update(activeBatch,false); 
+        if (followupJob != null) {
+            if(!Test.isRunningTest()) {                        
+                System.enqueueJob(followupJob);
+            }
+        }
+        System.debug(activeBatch);
```
Comment (cgijeffolsen) [inline TPL/force-app/main/default/classes/AccountQueueable.cls:32]: This system debug statement should be removed in later releases.

**Review by NataliaNikishina — APPROVED** [review]
@ahshaik1 approved
cc:  @crystalz2022 @rahulmantriSFDC

**Review by rahul-mantri — APPROVED** [review]
Approved @ahshaik1 

CC: @NataliaNikishina @crystalz2022

**Review by crystalz2022 — APPROVED** [review]
Approved @ahshaik1

**Pull request merged**